docs/agent-quickstart.md

```diff
@@ -108,6 +108,29 @@ $ fixie agent deploy
 
 This process takes about a minute to complete. Once deployed, you can use your agent via the [Fixie Web UI](http://app.fixie.ai) or the `fixie console` tool.
 
+## Agent REST API
+
+Once deployed (via `fixie agent serve` or `fixie agent deploy`), your agent can also be invoked
+via a REST API, using a `POST` request to the endpoint `/api/agent/<username>/<agent_handle>`.
+For example, the `fixie/calc` agent can be invoked as follows:
+
+```bash
+$ export FIXIE_API_KEY=<Your Fixie API key>
+$ curl https://app.fixie.ai/api/agents/fixie/calc \
+  -H "Authorization: Bearer ${FIXIE_API_TOKEN}" \
```

Comment:
This should be `FIXIE_API_KEY`, not `FIXIE_API_TOKEN`.

Reply:
Good catch!